scripts/database_migration/README.md

```diff
@@ -0,0 +1,53 @@
+# Script Manutenzione del DB per ambienti di Test.
+
+__N.B:__ gli script presenti in questa directory non possono essere eseguiti più di uno alla 
+volta e con una sola esecuzione con lo stesso filesystem. Questo perché i nomi dei file 
+temporanei non sono randomizzati. E potrebbero anche coincidere tra script differenti.
+
+
+## Script per il caricamento delle configurazioni
+- Per le tabelle pn-PaperCap, pn-PaperZone, pn-PaperCost, pn-PaperDeliveryDriver, 
+  pn-PaperDeliveryFile, pn-PaperTender, rifarsi al microservizio pn-paper-channel. 
+  Cartella _pn-paper-channel/scripts/aws/migration_ per quanto riguarda le prime due 
+  tabelle mentre per le altre tabelle si usa la funzionalità di BackOffice 
+  _Gestione gare__
+- Le tabelle pn-aggregates, pn-apiKey, pn-paAggregations, non sono ricopiabili e vanno 
+  rigenerate tramite chiamate rest, eventualmente eseguite tramite funzionalità di test 
```

Comment:
@marco-vit-pagopa manca la copia dell'api key e dei dati contenuti in selfcare (associazione nuovo ambiente PN con le PA, associazione dei diritti di accesso ai nuovi ambienti, duplicare gruppi da un ambiente all'altro).